README.md

```diff
@@ -65,7 +65,7 @@ class MyComponent extends React.Component {
 
 MyComponent.propTypes = {
   // You can declare that a prop is a specific JS primitive. By default, these
-  // are all optional.
+  // are all optional - `undefined` and `null` are allowed.
```

Comment:
in addition, they can be omitted entirely (which is distinct from being present but `undefined`). let's note that too!